README.md

```diff
@@ -96,6 +97,37 @@ class SomeService::SomeResource < Served::Resource::Base
 end
 ```
 
+## JsonAPI 
+
+Served does support JSON API responses and comes with a dedicated serializer. Nested resources are supported as well. 
+By default `Served` is raising exceptions on error.
+
+```ruby
+class JsonApiResource < Served::Resource::Base
+  def self.serializer
+    Served::Serializers::JsonApi
+  end
```

Comment:
you can do this by calling `serializer Served::Serializers::JsonApi` rather than overriding